src/utils/notify-screen-reader.ts

```diff
@@ -0,0 +1,13 @@
+let notifyAlert: HTMLElement;
+export function notifyScreenReader(text: string) {
+    if (!notifyAlert) {
+        notifyAlert = document.createElement('p');
+        notifyAlert.style.position = 'absolute';
+        notifyAlert.style.top = '-100px';
```

Comment:
it could be visible in some default font cases, better use `-100%`

Reply:
or better `bottom: 100%`